### DISCUSSION TITLE
Internationalisation

### DISCUSSION THREAD

**Discussion by ArvinZJC** [ArvinZJC/PaimonTray#21] [Announcements]
With the release of V1.1-stable.1, contributors to the app's internationalisation are greatly welcome. Please accept ❤ in appreciation of your translations in [Crowdin](https://crowdin.com/project/paimontray).

**Comment by comradekingu** [comment]
Happy to help if you use Weblate.

**Reply by ArvinZJC** [reply]
After consideration, I will keep using Crowdin only. Thanks anyway!
